css-view-transitions-1/Overview.bs

```diff
@@ -468,6 +500,16 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 			Note: This is used to hold dynamic styles relating to transitions.
 	</dl>
 
+## Additions to [=document element=] ## {#additions-to-document-element}
+
+	A [=document element=] additionally has a <dfn for="document element">view-transition root pseudo-element</dfn>.
+	A ''::view-transition'' or null. Initially null.
+
+	When this is a ''::view-transition'', the ''::view-transition'' renders as a child of the [=document element=],
+	and the [=document element=] is its [=originating element=].
+
+	Note: The position of the ''::view-transition'' within the [=document element=] does not matter, as the ''::view-transition'''s [=containing block=] is the [=snapshot viewport=].
```

Comment:
@khushalsagar is this a correct statement?

Reply:
> A [=document element=] additionally has a <dfn for="document element">view-transition root pseudo-element</dfn>.
  A ''::view-transition'' or null. Initially null.
  When this is a ''::view-transition'', the ''::view-transition'' renders as a child of the [=document element=],
  and the [=document element=] is its [=originating element=].
	
Do we need all this text? I figured its enough to say: [=document element=] is the [=originating element=] for ''::view-transition'' and ''::view-transition'''s [=containing block=] is the [=snapshot viewport=].

What we do need is spec text to say:

- ''::view-transition'' is not clipped by [=document element=].
- Effects on the [=document element=] do not apply to ''::view-transition''.

There is similar spec text for this for top layer elements [here](https://fullscreen.spec.whatwg.org/#new-stacking-layer:~:text=Ancestor%20elements%20with%20overflow%2C%20opacity%2C%20masks%2C%20etc.%20cannot%20affect%20it.).

''::view-transition'' renders as a child of the [=document element=] makes it sound like its the opposite of the above 2 points.

Reply:
I want a thing that can be `::view-transition` or null, so when it's not null, it's rendered, and when it's null, it's not rendered. As in, a way to say when it's in the document or not.

So… what is it 'in' when it's there?